src/blueprints/tv2/value-objects/tv2-a-b-source-layers.ts

```diff
@@ -0,0 +1,23 @@
+import { Tv2CasparCgLayer, Tv2SisyfosLayer } from './tv2-layers'
+
+interface Tv2ABSourceLayers {
+  caspar: {
+    clipPending: string
```

Comment:
It's the SourceLayer Blueprints gives the Server TimelineObjects at ingest.
Since we don't know which "MediaPlayer" is going to play the TimelineObject at ingest, we then can't assign the actual layer that the Server TimelineObject needs to be on (i.e. `casparcg_clip_player_1` and `casparcg_clip_player2`).

Instead, this "pending" layer is introduced that we can put Servers on at ingest that we then change on the TimelineObject when we do the A/B assignment.

Sidenote: This is taken directly from Blueprints.